BaseSpace.SDK/ServiceModels/Request/GetUserPermissionRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿
+
+namespace Illumina.BaseSpace.SDK.ServiceModels
+{
+    public class GetUserPermissionRequest : AbstractResourceRequest<GetUserPermissionResponse>
```

Comment:
perhaps rename this and the client method to something like GetAccessTokenDetails ?